posthog/migrations/0186_stale_events.py

```diff
@@ -0,0 +1,60 @@
+# Generated by Django 3.2.5 on 2021-11-30 08:23
+import django.utils.timezone
+from django.conf import settings
+from django.db import migrations, models
+
+from posthog.constants import AnalyticsDBMS
+
+
+def set_created_at(apps, schema_editor):
```

Comment:
Unsure whether this backfill is a good idea, it could potentially take a long time (though I don't think it'd freeze the table). I could also use some help testing the CH version as I have problems running it on my computer (opted not to waste time fixing it as the M1 is coming pretty soon).